### ISSUE TITLE
Increase Developer Productivity with Unix Bash Command One-Liners

### ISSUE THREAD

**Issue by fgantt** [fgantt/guides#3]
## TL;DR
Handy bash one-liners.

### Paper Link
https://itnext.io/increase-developer-productivity-with-unix-bash-command-one-liners-2973bccd7600

### Author/Institution
Lewis Fairweather

## Comments
Which process is using precious memory
$ ps aux | awk '{if ($5 != 0 ) print $2,$5,$6,$11}' | sort -k2n
Uses ps to output a sorted list of processes with the most memory usage.
Show 10 largest open files
$ lsof / | awk '{ if($7 > 1048576) print $7/1048576 "MB" " " $9 " " $1 }' | sort -n -u | tail
Uses lsof to list open files, sorted by size.
Print most common commands
$ cat ~/.bash_history | tr "\|\;" "\n" | sed -e "s/^ //g" | cut -d " " -f 1 | sort | uniq -c | sort -n | tail -n 15
